### ISSUE TITLE
Tabs on Android are hidden and showing up 

### ISSUE THREAD

**Issue by AlmogRnD** [aksonov/react-native-router-flux#2286]
Versoin 
"react": "16.0.0-alpha.12",
 "react-native": "0.47.1",
 "react-native-router-flux": "^4.0.0-beta.18",

Attached screenshots as you can see in Andiord tabs navigation not showing up correlty 

<img width="454" alt="screen shot 2017-08-24 at 10 29 35 am" src="https://user-images.githubusercontent.com/169507/29679779-6b16b004-88b7-11e7-80a2-0c10cacc788c.png">

<img width="411" alt="screen shot 2017-08-24 at 10 29 16 am" src="https://user-images.githubusercontent.com/169507/29679783-6de695b0-88b7-11e7-948d-1cc1938e8510.png">

My code
```
<View style={styles.container}>
                <MessageBar/>

                <Provider store={store}>
                    <ConnectedRouter navigationBarStyle={{backgroundColor:'#25abf9'}} titleStyle={{color:'white'}} headerTintColor="#FFF">
                        <Scene key="root" hideNavBar hideTabBar>
                            <Scene key="public">
                                <Scene key="login" component={Login} initial={true} title={'Login'} />
                                <Scene key="ResetPassword" component={ResetPassword} title={'Reset Password'} />
                            </Scene>
                            <Drawer contentComponent={CustomDrawer}>
                            <Scene key="tabBar" tabs lazy showLabel={false} inactiveBackgroundColor={'#30393F'} activeBackgroundColor={'#25abf9'}>
                                <Scene key="tab1" title="Home" tabBarLabel="Home" icon={HomeTabIcon}>
                                    <Scene key='Home' component={Home} title='Home'  initial={true}  panHandlers={null} />
                                    <Scene key='Teams' component={Teams} title={'Teams'} back initial={false} renderLeftButton={()=>{}}  />
                                    <Scene key='Sync' component={Sync} title='Syncing Data' initial={false}  hideTabBar hideNavBar />
                                </Scene>
                                <Scene key="tab2" title="Photos" tabBarLabel="" icon={PhotosTabIcon}>
                                    <Scene key='Photos' component={Photos} title={'Photos'} back renderLeftButton={()=>{}} backTitle='' backButtonTextStyle={{color:'white'}} initial={false} />
                                    <Scene key='SinglePhoto' component={SinglePhoto} title={'Photo'} renderLeftButton={()=>{}} back backTitle='' backButtonTextStyle={{color:'white'}} initial={false} />
                                    <Scene key='PhotoUpload' component={PhotoUpload} title='Upload Photo' renderLeftButton={()=>{}} back backTitle='' backButtonTextStyle={{color:'white'}} initial={false} />
                                </Scene>
                            </Scene>
                            </Drawer>
                        </Scene>
                    </ConnectedRouter>
                </Provider>

            </View> 
```

**Comment by aksonov** [comment]
Closed due #2288 (bug in more recent beta.19)

**Comment by AlmogRnD** [comment]
I was able to get it working, need to set icon to true
